src/components/shows/showviewer.js

```diff
@@ -0,0 +1,194 @@
+import React from "react"
```

Comment:
Some general design things to think about:

* When viewing a show, there is no back/close button, only next and previous show buttons, on all but the most and least recent shows. A back/close button should always be present in the UI somewhere when viewing a "detail" screen so that the user doesn't feel trapped or stuck.
![image](https://user-images.githubusercontent.com/1442037/79944801-906ef700-84c0-11ea-9c2c-af7107093155.png)

* Extending from the previous point, you might want to try and store the user's viewed shows in the browser's Back history so that the browser Back button can go back. That would probably be really hard though so worry about it later.

* The main viewer makes heavy use of hover-over text elements. These are cool, but less accessible and notably people on phones/touchscreen devices can't see them at all. This doesn't affect usability too much (the posters have `alt` text which blind people can access) but is worth thinking about.

* The hover-over transition speed is also a bit slow at 0.75 seconds. I usually prefer closer to 300ms so that it feels nice and snappy.

Reply:
All done! :)